spring-cloud-gcp-data-firestore/src/main/java/com/google/cloud/spring/data/firestore/FirestoreTemplate.java

```diff
@@ -291,10 +301,17 @@ public FirestoreMappingContext getMappingContext() {
 				//In a transaction, all write operations should be sent in the commit request, so we just collect them
 				return Flux.from(documentNames).doOnNext(t -> writes.add(createDeleteWrite(t)));
 			}
-			return ObservableReactiveUtil.streamingBidirectionalCall(
-					this::openWriteStream,
-					documentNames.bufferTimeout(this.writeBufferSize, this.writeBufferTimeout),
-					this::buildDeleteRequest);
+			return Flux.from(documentNames).bufferTimeout(this.writeBufferSize, this.writeBufferTimeout)
+					.flatMap(ts -> {
```

Comment:
Same, rename `ts` to something -- it also turns into `e` for Entity later in this block, so maybe something more consistent like "entity".